scala/build/Dockerfile.latest

```
@@ -0,0 +1,151 @@
+
+# TODO - update to yarn 2
+# https://yarnpkg.com/getting-started/install
+
+# Install AWS CLI
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && \
+    unzip awscliv2.zip && \
+    ./aws/install && \
+    rm -rf awscliv2.zip ./aws
+
+# Install Docker and Docker Compose
+RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - && \
```

Hadolint

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)